frontendR2.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html lang = 'en'>
+  <head>
+    <meta charset = "utf-8">
+    <meta name = "description" content = "a page for entering your location">
+    <title> Shorts Page </title>
+      <style>
+      html {
+      margin: 0%;
+      padding: 0%;
+      background-color: #19c5ff;
+      }
+      h1 {
+      color: white;
+      font-family: Georgia, "Times New Roman";
+      font-size: 3em;
+      }
+      h2 {
+      font-size: 1em;
+      font-family: Georgia;
+      }
+      p {
+      color: black;
+      font-size: 1em;
+      }
+      body {
+      width: 23%;
+      margin: 0 auto;
+      text-align: justify;
+      font-family: Helvetica, Arial, serif;
+      }
+      </style>
+  </head>
+  <body>
+    <h1>The Shortist</h1>
+    <h2>Should you wear shorts or nah?</h2>
+
+    <p>
+      We'll keep this short:
+      <br> 1. Enter your zipcode
+      <br> 2. Find out if it's cool to wear shorts today
+      <br> 3. Chillax
+    </p>
+
+      <form method= "post">
+        <div id="zipform">
+        <input type="text" id="zipcode">
+        </div>
+      </form>
+
+      <div class = "button">
+        <button type = "submit" id="setzipcode">Set Zipcode</button>
+      <br>
+      </div>
+
+      <p>
+      <div id="display">
+      </div>
+      </p>
+
+<script type="text/javascript">
+//Return API weather input
+
+//get zipcode value
+  var zip = "'"+ document.getElementById("zipform").getElementById("zipcode").value +"'";
+//get weather from API
+  var weather = new XMLHttpRequest();
+  weather.open("GET", "https://api.apixu.com/v1/current.json?key=3844afb51b4b4d96840161310172608&q=" + zip, false);
```

Comment:
Why don't you try putting this in its own function?

Have a function that is 
`apiGet(zip)`
Where you pass in the zip. That way you can reuse it later

Reply:
Yeah I thought about doing that but wasn't sure if I would need to use it again. but I guess either way it's good to have in a function because it's not too much extra work to get it in a function and then you can call it whenever